apps/quickstart-integrations-monday-code/src/middleware.js

```diff
@@ -12,7 +12,17 @@ export const authorizeRequest = (req, res, next) => {
     if (!authorization && req.query) {
       authorization = req.query.token;
     }
+    if (typeof authorization !== "string") {
+      logger.info("No credentials in request");
+      res.status(401).json({ error: "Not authenticated, no credentials in request" });
+      return;
```

Comment:
Remove the empty return, not necessary
```suggestion

```

Reply:
hmm - it's a copy from:
https://github.com/mondaycom/welcome-apps/blame/0fe9338fc56a145629bbb4de07cbc9865702cb1b/apps/quickstart-integrations-ts/src/middlewares/authentication.ts#L30
I'm not sure whether the two code sections shouldn't be the same. Otherwise it confuses other developers who look at both files, doesn't it?

Reply:
Ah, I didn't realize it's in there too. I'll delete the returns from the other example; we don't need to return anything when using `res.status().send()`

Reply:
EDIT: But I guess in that one typescript complains if you don't have a return value...